requirements.txt

```diff
@@ -0,0 +1,10 @@
+six
+lxml
+Flask==0.10.1
+Flask-Admin==1.4.0
+Flask-SQLAlchemy==2.1
+Flask-WTF==0.12
+WTForms==2.1
+scrapy==1.1.0rc3
+formasaurus[with-deps]==0.7.1
+-e git+git@github.com:lopuhin/scrapy-splash.git@8c08ad60d1668a9160c63c32801a04e49c5671b2#egg=scrapyjs
```

Comment:
version from master branch should be fine now, as https://github.com/scrapy-plugins/scrapy-splash/pull/48 is merged